README.md

```diff
@@ -11,17 +11,78 @@ Analyze, Annotate, Publish. Turn documents into data.
 4. [invoke][invoke-install]
 
-### Installation Steps
+### Installation of DocumentCloud and its Authentication System
 
-1. Check out the git repository - `git clone git@github.com:MuckRock/documentcloud.git`
-2. Enter the directory - `cd documentcloud`
-3. Run the dotenv initialization script - `python initialize_dotenvs.py`
+1. Install software above and Git Large File support [using these instructions](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage). 
+   - Ensure you have at least an additional 11 gigabytes of hard disk space allocated to Docker for these purposes.
+   - Ensure your Docker host application has at least 7gb of memory allocated, 10gb preferred. 
+   - These instructions create 3 distinct docker-compose sessions, with the Squarelet session hosting the shared central network. 
+2. Check out the git repository - `git clone git@github.com:MuckRock/documentcloud.git`
+3. Enter the directory - `cd documentcloud`
+4. Run the dotenv initialization script - `python initialize_dotenvs.py`
    This will create files with the environment variables needed to run the development environment.
-4. Start the docker images - `inv up`
-   This will build and start all of the docker images using docker-compose. It will bind to port 80 on localhost, so you must not have anything else running on port 80. The invoke tasks specify the `local.yml` configuration file for docker-compose. If you would like to run docker-compose commands directly, set the environment variable `export COMPOSE_FILE=local.yml`.
-5. Set `api.dev.documentcloud.org` and `minio.documentcloud.org` to point to localhost - `sudo echo "127.0.0.1 api.dev.documentcloud.org minio.documentcloud.org" >> /etc/hosts`
-6. Enter `api.dev.documentcloud.org/` into your browser - you should see the Django API root page.
-7. Install and run [Squarelet](https://github.com/muckrock/squarelet) and the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) following the instructions in their repos to view the full-stack application.
+5. You do not need to immediately start the docker-compose session for DocumentCloud until the Squarelet authentication service integration below is complete.
+6. Set `api.dev.documentcloud.org` and `minio.documentcloud.org` to point to localhost - `sudo echo "127.0.0.1 api.dev.documentcloud.org minio.documentcloud.org" >> /etc/hosts`
+7. Enter `api.dev.documentcloud.org/` into your browser - you should see the Django API root page. **Note that `api` is before `dev` in this service URL.**
+8. Install and run [Squarelet](https://github.com/muckrock/squarelet) and the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) following the instructions in their 2 repos as part of the full-stack application. 
+    - **Create Squarelet user:** 
+      - `inv sh` (not shell) then `./manage.py createsuperuser` in Squarelet (not DocumentCloud).
```

Comment:
These instructions should probably be in the Squarelet readme - also I think it is simpler to do `inv manage createsuperuser`

Reply:
Fixed in next commit